baseplate/clients/sqlalchemy.py

```diff
@@ -156,30 +159,50 @@ class SQLAlchemyEngineContextFactory(ContextFactory):
 
     """
 
-    PROM_PREFIX = "bp_sqlalchemy_pool"
-    PROM_LABELS = ["pool"]
+    PROM_PREFIX = "databasebp"
```

Comment:
Hm, it could be sqlite or another dialect, so I'd rather keep it generic. `bp` tacked on because that's what baseplate.go does. 

If the dialect is interesting I'd rather have dialect as a label, since that makes automating dashboards easier.

Reply:
hm... I feel like `baseplate_db_...` or `baseplate_database_...` or `baseplatedb_...` even would make more sense as a prefix. But if baseplate.go already made that decision for us...

Reply:
@krav we shouldn't be merging them if the metric name is incorrect, I think a release is too late.

With regards to the prefix, lets go with `sql` based on the current spec iteration